test/unit/lookups/olleh_test.rb

```diff
@@ -225,6 +225,16 @@ def test_olleh_route_searching_with_waypoints
   end
 
   def test_olleh_addr_step_search
+    # VCR.use_cassette('km2_local_search') do
+      query = Geocoder::Query.new('분당구+689', {places: 5})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use 2 (not 4) spaces for indentation.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.